app/assets/javascripts/components/static/about_page.es6.jsx

```diff
@@ -2,69 +2,85 @@ class AboutPage extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = {
-      people: [
-        'daniel', 'richard', 'rocky',
-        'rachel', 'sahil', 'lancy'
-      ],
-    }
-  }
+    this.individuals = [
```

Comment:
you should either move this data into a table or maybe some json file within the repo just so this file isnt cluttered with this hardcoded stuff, and if someone wanted to change the info itd also be easier to have it not in this file